src/components/Navbar/index.js

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from "react";
 import { FaBars } from "react-icons/fa";
 import { IconContext } from "react-icons/lib";
 import { animateScroll as scroll } from "react-scroll";
+import resume from "../../Files/MatthewBeckermanResumeSP2022.pdf";
```

Comment:
Instead of `Files`, call the folder `assets`

Reply:
Should I restructure the other folders as well? Currently have separate folders for my images and videos

Reply:
Yes. One `assets` folder. And within it you can separate it out into images, videos or just top level. However you like.

Reply:
Gotcha, so then do I close this pull request and then create a new one after I've made the changes?

Reply:
Nope. When you push up an update it will get added to this PR. It becomes like a history for this feature.

Reply:
Ahhhh, that's pretty neat